.github/workflows/release.yml

```diff
@@ -168,7 +168,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [16.x]
```

Comment:
Shouldn't this be `16.x, 14.x, 12.x`?

Reply:
No, we only run e2e tests in one environment. Unit tests, though, are run everywhere.